### PR TITLE
add Dave's custom item

### PR DESCRIPTION
**Pull request by Casper3667**
https://baystation12.net/forums/threads/small-photograph-dayton-cosgrove-your_neighbor_dave.3931/

### CODE REVIEW

File: config/custom_items.txt [hunk 1]
```diff
@@ -629,4 +629,12 @@ item_path: /obj/item/clothing/glasses/hud/health
 item_name: Retinal Health Scanner
 item_icon: vichud
 item_desc: A small medical scanner similar to the one used in medical HUDs. This one has been designed to connect to and display the information directly on the wearer's implanted retinal overlay. The device includes a simple projector for displaying the data for users with unaugmented eyes.
-}
\ No newline at end of file
+}
+{
+ckey: your_neighbor_dave
```
Comment (Casper3667) [inline config/custom_items.txt]: the name is causing issues, is it the _ ? We might need to fix that...

Comment (Kasuobes) [reply config/custom_items.txt]: There are no underscores in ckeys.

Comment (Casper3667) [reply config/custom_items.txt]: uh, yes there is: http://i.imgur.com/tNQUYNU.png

Comment (Casper3667) [reply config/custom_items.txt]: or do you mean to just type it in the name without underscores?

Comment (Datraen) [reply config/custom_items.txt]: ckeys are normalized, so the underscore does not exist in this context. Hence why Broseph Stylin turns into brosephstylin (example on line 543)

Comment (Casper3667) [reply config/custom_items.txt]: hmmm, I guess I will do that then, thanks.

**Pull request merged**